content/quickstart/java/metrics.md

```diff
@@ -856,12 +867,9 @@ import java.util.List;
 import io.opencensus.stats.Aggregation;
 import io.opencensus.stats.Aggregation.Distribution;
 import io.opencensus.stats.BucketBoundaries;
-import io.opencensus.stats.Stats;
```

Comment:
Why did we remove these imports?

Reply:
Some code snippets introduce an import that already exists

Reply:
But this snippet doesn't, right? Perhaps for brevity of this PR, let's revert the imports and send smaller PRs specialized for PRs otherwise we might spend more time on Java imports and neglect the rest of the gist of the PR :)

Reply:
Thank you for the clarification offline @hvent90! The specialized snippet that you've edited is meant to introduce new imports and not ones that already exist.